src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -112,7 +112,14 @@ public static void Initialize(Win32PlatformOptions options)
                 .Bind<IRenderTimer>().ToConstant(new DefaultRenderTimer(60))
                 .Bind<ISystemDialogImpl>().ToSingleton<SystemDialogImpl>()
                 .Bind<IWindowingPlatform>().ToConstant(s_instance)
-                .Bind<PlatformHotkeyConfiguration>().ToSingleton<PlatformHotkeyConfiguration>()
+                .Bind<PlatformHotkeyConfiguration>().ToConstant(new PlatformHotkeyConfiguration(KeyModifiers.Control)
+                {
+                    OpenContextMenu =
+                    {
+                        // Add Shift+F10
+                        new KeyGesture(Key.F10, KeyModifiers.Shift)
```

Comment:
Seems to be only Windows specific gesture

Reply:
See above comment about `WM_CONTEXTMENU`.